From 017a2630e64a0a2398a72d7b2e16598564ca35d5 Mon Sep 17 00:00:00 2001
From: Dan Williams <dan.j.williams@intel.com>
Date: Thu, 17 Mar 2011 08:04:43 -0700
Subject: [PATCH 067/479] isci: task.h compile and checkpatch fixes

upstream: ce0b89f35636f3a69fbde851aff8a284c4b4c22b

A usage of "FALSE" leaked in as well as some checkpatch escapes.

Signed-off-by: Dan Williams <dan.j.williams@intel.com>

Integrated-by: Hui Wang <Hui.Wang@windriver.com>
---
 drivers/scsi/isci/task.h |   42 +++++++++++++++++++-----------------------
 1 files changed, 19 insertions(+), 23 deletions(-)

diff --git a/drivers/scsi/isci/task.h b/drivers/scsi/isci/task.h
index b84bedd..9754b43 100644
--- a/drivers/scsi/isci/task.h
+++ b/drivers/scsi/isci/task.h
@@ -306,7 +306,6 @@ isci_task_set_completion_status(
 	* is in the error path.
 	*/
 	if (task->task_state_flags & SAS_TASK_NEED_DEV_RESET) {
-
 		/* Fail the I/O to make sure it goes into the error path. */
 		response = SAS_TASK_UNDELIVERED;
 		status = SAM_STAT_TASK_ABORTED;
@@ -317,28 +316,25 @@ isci_task_set_completion_status(
 	task->task_status.stat = status;
 
 	switch (task_notification_selection) {
-
-		case isci_perform_aborted_io_completion:
-			/* This path can occur with task-managed requests as well as
-			* requests terminated because of LUN or device resets.
-			*/
-			/* Fall through to the normal case... */
-
-		case isci_perform_normal_io_completion:
-			/* Normal notification (task_done) */
-			isci_set_task_doneflags(task);
-			break;
-
-		default:
-			WARN_ON(FALSE);
-			/* Fall through to the error case... */
-
-		case isci_perform_error_io_completion:
-			/* Use sas_task_abort */
-			/* Leave SAS_TASK_STATE_DONE clear
-			*  Leave SAS_TASK_AT_INITIATOR set.
-			*/
-			break;
+	case isci_perform_aborted_io_completion:
+		/* This path can occur with task-managed requests as well as
+		 * requests terminated because of LUN or device resets.
+		 */
+		/* Fall through to the normal case... */
+	case isci_perform_normal_io_completion:
+		/* Normal notification (task_done) */
+		isci_set_task_doneflags(task);
+		break;
+	default:
+		WARN_ONCE(1, "unknown task_notification_selection: %d\n",
+			 task_notification_selection);
+		/* Fall through to the error case... */
+	case isci_perform_error_io_completion:
+		/* Use sas_task_abort */
+		/* Leave SAS_TASK_STATE_DONE clear
+		 * Leave SAS_TASK_AT_INITIATOR set.
+		 */
+		break;
 	}
 
 	spin_unlock_irqrestore(&task->task_state_lock, flags);
-- 
1.7.0

